charts/emissary-ingress/templates/_helpers.tpl

```diff
@@ -71,7 +71,7 @@ disabled if fullImageOverride is present
 {{- if hasKey .Values.image "repository" -}}
 {{- printf "%s:%s" .Values.image.repository .Values.canary.image.tag -}}
 {{- else -}}
-{{- printf "%s:%s" "docker.io/emissaryingress/emissary" .Values.canary.image.tag -}}
+{{- printf "%s:%s" "docker.io/ambassador/ambassador-agent" .Values.canary.image.tag -}}
```

Comment:
These changes are problematic because it'll also use that image for the main Emissary.  It needs to grow a separate image config for the agent.

Reply:
the helm chart is updated